docs/API.md

```diff
@@ -979,26 +979,26 @@ Example response:
 
 An object with the following properties:
 
-* `"username"`: An *username*. The name of the account of an authorized user.
-* `"api_key"`: An *api key*. The api_key associated with the username.
-* `"domains"`: A list of *domain names*. The domains to be tested.
+* `"username"`: An *username*, required. The name of the account of an authorized user.
+* `"api_key"`: An *api key*, required. The api_key associated with the username.
+* `"domains"`: A list of *domain names*, required. The domains to be tested.
 * `"test_params"`: As described below.
 
 The value of `"test_params"` is an object with the following properties:
 
-* `"client_id"`: A *client id*, optional.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A *client version*, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A *client id*, optional. (default: `Zonemaster Batch Scheduler`)
```

Comment:
I'm not sure I understand the design here. Is this value supposed to be decided by the user or by the RPCAPI daemon? First the user's argument is validated, but then no matter what the user specified, the daemon forces the value to this string.

Same goes for `"client_version"`.

Reply:
I agree, BUT, i also updated the daemon to be able to use the clients value.
I change the default behavior to allow default value in the commit 83893bef7e7100e82c1fc014ffe0c55cd2de2d4f. 
I considered the past code as a bug, and so, when a removed the deprecated code, i fixed this bug.

Reply:
Ok. But in that case, do you really think `"Zonemaster Batch Scheduler"` is a sensible default value?
Also, please make sure to add quotation marks to string literals in this document. There are couple of `"default"` without quotation marks in this PR.

Reply:
done